usersync/usersyncers/syncer_test.go

```diff
@@ -35,28 +36,30 @@ func TestNewSyncerMap(t *testing.T) {
 			string(openrtb_ext.BidderIx):             syncConfig,
 			string(openrtb_ext.BidderLifestreet):     syncConfig,
 			string(openrtb_ext.BidderLockerDome):     syncConfig,
-			string(openrtb_ext.BidderMgid):           syncConfig,
-			string(openrtb_ext.BidderOpenx):          syncConfig,
-			string(openrtb_ext.BidderPubmatic):       syncConfig,
-			string(openrtb_ext.BidderPulsepoint):     syncConfig,
-			string(openrtb_ext.BidderRhythmone):      syncConfig,
-			string(openrtb_ext.BidderRTBHouse):       syncConfig,
-			string(openrtb_ext.BidderRubicon):        syncConfig,
-			string(openrtb_ext.BidderSharethrough):   syncConfig,
-			string(openrtb_ext.BidderSomoaudience):   syncConfig,
-			string(openrtb_ext.BidderSovrn):          syncConfig,
-			string(openrtb_ext.BidderSynacormedia):   syncConfig,
-			string(openrtb_ext.BidderTriplelift):     syncConfig,
-			string(openrtb_ext.Bidder33Across):       syncConfig,
-			string(openrtb_ext.BidderSonobi):         syncConfig,
-			string(openrtb_ext.BidderVrtcal):         syncConfig,
-			string(openrtb_ext.BidderYieldmo):        syncConfig,
-			string(openrtb_ext.BidderGamma):          syncConfig,
-			string(openrtb_ext.BidderVisx):           syncConfig,
-			string(openrtb_ext.BidderGamoshi):        syncConfig,
-			string(openrtb_ext.BidderUnruly):         syncConfig,
-			string(openrtb_ext.BidderVerizonMedia):   syncConfig,
-			string(openrtb_ext.BidderEngageBDR):      syncConfig,
+
```

Comment:
Can we remove this blank line?

Reply:
ok

Reply:
Blank line in line 38 has not been removed yet. Could you please do?

Reply:
not sure what's going on here...it's removed in my local repo, and everything is pushed.